aiven/client/cli.py

```diff
@@ -2515,7 +2515,7 @@ def service__task_create(self):
                 raise argx.UserError("--target-version is required for this operation")
             body = {
                 "task_type": self.args.operation,
-                "target_version": self.args.target_version,
+                "target_version": str(self.args.target_version),
```

Comment:
so is it like it will be an integer as type is set to integer and then converted to the string to work here? what happens if it is given a wrong version?

Reply:
thanks for clarifying it!